go.mod

```diff
@@ -35,12 +35,12 @@ require (
 	google.golang.org/grpc v1.51.0
 	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools v2.2.0+incompatible
-	helm.sh/helm/v3 v3.10.3
-	k8s.io/api v0.25.2
-	k8s.io/apimachinery v0.25.2
-	k8s.io/cli-runtime v0.25.2
-	k8s.io/client-go v0.25.2
-	k8s.io/klog/v2 v2.70.1
+	helm.sh/helm/v3 v3.11.1
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.0
+	k8s.io/cli-runtime v0.26.0
+	k8s.io/client-go v0.26.0
```

Comment:
Why do you require 0.26 and then replace with 0.25?

Reply:
I don't require 0.26 it `go mod tidy` includes it. 
However, with 0.26, we get 

```
# sigs.k8s.io/controller-runtime/pkg/cache
/Users/alexey/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.11.2/pkg/cache/informer_cache.go:144:9: cannot use i.Informer (variable of type "k8s.io/client-go/tools/cache".SharedIndexInformer) as type Informer in return statement:
	"k8s.io/client-go/tools/cache".SharedIndexInformer does not implement Informer (wrong type for AddEventHandler method)
		have AddEventHandler(handler "k8s.io/client-go/tools/cache".ResourceEventHandler) ("k8s.io/client-go/tools/cache".ResourceEventHandlerRegistration, error)
		want AddEventHandler(handler "k8s.io/client-go/tools/cache".ResourceEventHandler)
/Users/alexey/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.11.2/pkg/cache/informer_cache.go:158:9: cannot use i.Informer (variable of type "k8s.io/client-go/tools/cache".SharedIndexInformer) as type Informer in return statement:
	"k8s.io/client-go/tools/cache".SharedIndexInformer does not implement Informer (wrong type for AddEventHandler method)
		have AddEventHandler(handler "k8s.io/client-go/tools/cache".ResourceEventHandler) ("k8s.io/client-go/tools/cache".ResourceEventHandlerRegistration, error)
		want AddEventHandler(handler "k8s.io/client-go/tools/cache".ResourceEventHandler)
```
will return to the issue later